ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-ref.yaml

```diff
@@ -30,6 +30,7 @@ ref:
     default: "true"
     documentation: |-
       Default is true, which means clean up the pod/resource that kube-burner ocp created, you can set it to false to keep the resouce
+  timeout: 4h
```

Comment:
I copied from another example in #forum-test-platform that had this. Hope it works.